src/MICore/Transports/TcpTransport.cs

```diff
@@ -61,7 +61,7 @@ public override void InitStreams(LaunchOptions options, out StreamReader reader,
                     null /*UserCertificateSelectionCallback */
                     );
 
-                sslStream.AuthenticateAsClientAsync(tcpOptions.Hostname, certStore.Certificates, System.Security.Authentication.SslProtocols.Tls, false /* checkCertificateRevocation */).Wait();
+                sslStream.AuthenticateAsClientAsync(tcpOptions.Hostname, certStore.Certificates, System.Security.Authentication.SslProtocols.None, false /* checkCertificateRevocation */).Wait();
```

Comment:
Is changing from Tls to None ok? It was requiring it to make it determined by the OS.

Reply:
Starting with .NET Framework 4.7, this method authenticates using None, which allows the operating system to choose the best protocol to use, and to block protocols that are not secure. In .NET Framework 4.6 (and .NET Framework 4.5 with the latest security patches installed), the allowed TLS/SSL protocols versions are 1.2, 1.1, and 1.0 (unless you disable strong cryptography by editing the Windows Registry).

Reply:
I'd #def and say when you are using 4.7.2 (IsXplat false?) then use none.